backend/src/services/game/hongKongGame.service.ts

```diff
@@ -67,10 +66,12 @@ class HongKongGameService extends GameService {
         });
     }
 
-    public getCurrentGames(): Promise<any[]> {
+    public getGames(filter: GameFilterArgs): Promise<any[]> {
+        const whereQuery = generateGameQuery(filter);
+
         return prisma.hongKongGame.findMany({
             where: {
-                status: GameStatus.IN_PROGRESS,
+                ...whereQuery,
             },
```

Comment:
think this can just be 
`return prisma.hongKongGame.findMany({
            where: whereQuery,`
            instead